lib/plugins/http-error-handler.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+function httpErrorHandler(request, reply) {
+  const response = request.response;
+  if (response.isBoom) {
+    let msg;
+    let statusCode = response.output.payload.statusCode;
+    let errorTitle = response.output.payload.error;
+    let errorMessage = response.output.payload.message;
```

Comment:
Agreed! I'd expect eslint to complain about it. Maybe it's somehow related to the new `lib/plugins` folder?